soc/espressif/esp32s3/memory.h

```diff
@@ -34,7 +34,7 @@
 /* The offset between Dbus and Ibus.
  * Used to convert between 0x403xxxxx and 0x3fcxxxxx addresses.
  */
-#define IRAM_DRAM_OFFSET         0x6f0000
+#define IRAM_DRAM_OFFSET         (SRAM1_IRAM_START - SRAM1_DRAM_START)
```

Comment:
same here. It is much more useful if the user can see the number and can operate with the offset directly..